content/03.results-and-discussion.md

```diff
@@ -182,16 +180,16 @@ includes schematic images of best practices or expected output that are
 published under CC-BY licenses to allow for use in other training material
 (example in {@fig:doc-image-example}). We hope this open-access resource will
 make the study of aDNA more accessible to researchers new to the field, by
-providing practical guidelines how to evaluate characteristics and effects of
-aDNA on downstream analyses.
+providing practical guidelines on how to evaluate the characteristics of aDNA 
+and their effects on downstream analyses.
 
 ![Example of output interpretation documentation schematic images that can
 assist new users in the interpretation to next-generation-sequencing aDNA
 processing.](images/Fig2_fastqc_adapter_content.png){#fig:doc-image-example
 width="70%"}
 
 The development of nf-core/eager in Nextflow and the nf-core initiative will
-also improve open-source community contributions to the pipeline. While Nextflow
+also allow for open-source development, while ensuring the high quality of community contributions to the pipeline. While Nextflow
```

Comment:
EAGER1 was already open source? 
```suggestion
also improve open-source development, while ensuring the high quality of community contributions to the pipeline. While Nextflow
```

Reply:
It is still something that development in Nextflow and nf-core contribute to, even if it isn;t a change from eager 1, no?

Reply:
oh wait. your suggestion works